php/class-fieldmanager-datepicker.php

```diff
@@ -93,23 +101,26 @@ public function form_element( $value ) {
 		ob_start();
 		include fieldmanager_get_template( 'datepicker' );
 
-		// Reset the timestamp
+		// Reset the timestamp.
 		$value = $old_value;
 		return ob_get_clean();
 	}
 
 	/**
-	 * Convert date to timestamp
-	 * @param $value
-	 * @param $current_value
-	 * @return int unix timestamp
+	 * Convert date to timestamp.
+	 *
+	 * @param mixed $value         The new value for the field.
+	 * @param mixed $current_value The current value for the field.
+	 * @return int Unix timestamp.
 	 */
 	public function presave( $value, $current_value = array() ) {
 		$time_to_parse = sanitize_text_field( $value['date'] );
 		if ( isset( $value['hour'] ) && is_numeric( $value['hour'] ) && $this->use_time ) {
 			$hour = intval( $value['hour'] );
 			$minute = ( isset( $value['minute'] ) && is_numeric( $value['minute'] ) ) ? intval( $value['minute'] ) : 0;
-			if ( $hour == 0 && $this->use_am_pm ) $hour = 12;
+			if ( 0 == $hour && $this->use_am_pm ) {
```

Comment:
Possibly, but I don't want to risk changing the field's behavior in this PR, which is intended only to make the field standards-compliant.